src/docfx/build/Build.cs

```diff
@@ -52,6 +52,7 @@ private static bool BuildDocset(string docsetPath, string? outputPath, CommandLi
                 }
 
                 errorLog.Configure(config, buildOptions.OutputPath);
+                new OpsPreProcessor(config, buildOptions).Run();
```

Comment:
nit: This could be a static method if it not put into context?
```suggestion
                OpsPreProcessor.Run(config, buildOptions);
```

Reply:
would prefer to follow the constructor injection